packages/legend-application-studio/src/stores/StoreRelational_LegendStudioApplicationPlugin_Extension.ts

```diff
@@ -97,4 +114,29 @@ export interface StoreRelational_LegendStudioApplicationPlugin_Extension
    * Get the list of renderers for the editor for a relational database authentication strategy.
    */
   getExtraAuthenticationStrategyEditorRenderers?(): AuthenticationStrategyEditorRenderer[];
+
+  // --------------------- relational database connection post processor ------------------
+
+  /**
+   * Get the list of the supported type specifiers for relational post processors
```

Comment:
--> `Get the list of post-processor type specifiers`. Could we change all `post processor` to `post-processor`?